generated/nifake/session.py

```diff
@@ -635,6 +637,20 @@ def get_array_using_ivi_dance(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return [float(array_out_ctype[i]) for i in range(array_size_ctype.value)]
 
+    def get_custom_type(self):
+        '''get_custom_type
+
+        This function takes a custom type.
+
+        Returns:
+            cs (custom_struct): Set using custom type
+        '''
+        vi_ctype = visatype.ViSession(self._vi)  # case 1
+        cs_ctype = custom_struct.CustomStruct()  # case 13
+        error_code = self._library.niFake_GetCustomType(vi_ctype, ctypes.pointer(cs_ctype))
+        errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
+        return custom_struct.custom_struct(cs_ctype)
```

Comment:
If I understand this correctly, custom_struct.CustomStruct is the ctypes object and custom_struct.custom_struct is the object clients know about.

custom_struct.CustomStruct is an implementation detail... does it leak to the client?

Reply:
I renamed the ctypes implementation to `_CustomStructCType` and the Python type to `CustomStruct`.

I also created `custom_struct` that is the same as the ctype. I needed the CamelCase name to make flake8 happy (unless we use noqa) but snake case to match the API definition. This is what NI-SCOPE will look like where the API type is `wfm_info`

The client will interact only with `CustomStruct`.